xgcm/grid.py

```diff
@@ -1359,6 +1361,26 @@ def iterate_axis_combinations(items):
         # return the product of the metrics
         return functools.reduce(operator.mul, metric_vars, 1)
 
+    def interp_like(self, da_source, metric):
+        for axname, axis in self.axes.items():
+            try:
+                position_da, _ = axis._get_axis_coord(da_source)
+                position_metric, _ = axis._get_axis_coord(metric)
+            except KeyError:
+                continue
+            if not position_da == position_metric:
```

Comment:
```suggestion
            if position_da != position_metric:
```

minor comment: slightly more conventional syntax